kinetic/distribution.yaml

```diff
@@ -1774,11 +1774,10 @@ repositories:
       - cob_moveit_bringup
       - cob_moveit_interface
       - cob_obstacle_distance_moveit
-      - cob_pick_place_action
```

Comment:
just to confirm:
I don't want to release a new version of `cob_pick_place_action`
If `cob_pick_place_action` is no longer a dependency of `cob_manipulation` meta-package - that is fine
I still want `cob_pick_place_action` to be available under the old version `0.7.2-1`

Reply:
I don't know if it will work like that.  My understanding was that a sync more-or-less deleted everything, then just uploaded the new versions of packages.  But I'm not 100% sure of that.  @nuclearsandwich @tfoote , can you confirm?

Reply:
still waiting for confirmation/clarification on the effect of this pr...

Reply:
Removing the package here will remove it from the buildfarm. Packages in the same repository cannot target be different versions or target different tags. Options are:

1. Continue releasing `cob_pick_place_action` in this repo with no changes in each version.
2. Split the package out into a separate repository and release it once from there, it can be the same source version as is currently released you just need to bump the package revision. Then you can release this repository without affecting it.

Reply:
Ok, then I'm ok with dropping `cob_pick_place_action` package from buildfarm for `kinetic` - which is what this PR is doing
Once I find the time to fix it again, I will include it again and trigger a new release that will make the package available via buildfarm again
:point_right: good to merge